src/blob_gc_picker.cc

```diff
@@ -37,17 +44,42 @@ std::unique_ptr<BlobGC> BasicBlobGCPicker::PickBlobGC(
                      blob_file->file_number());
       continue;
     }
-    blob_files.push_back(blob_file.get());
 
-    batch_size += blob_file->file_size();
-    if (batch_size >= cf_options_.max_gc_batch_size) break;
+    if (!stop_picking) {
+      blob_files.push_back(blob_file.get());
+      batch_size += blob_file->file_size();
+      estimate_output_size +=
+          (blob_file->file_size() - blob_file->discardable_size());
+      if (batch_size >= cf_options_.max_gc_batch_size &&
```

Comment:
it should be `||` ?

Reply:
I think gc_picker stops picking file when the batch_size greater than max_batch_size and output_size is larger than min_output_threshold, to prevent generation of small files, so if && changed to || it would be same as the origin version...(but the strategy maybe too simple, too)

Reply:
If the output size always small, it would lead to reading too much in a GC round.

Reply:
@GlitterIsMe

Reply:
@yiwu-arbug What do you think?